docs/shared/make_hgrid.yaml

```diff
@@ -0,0 +1,26 @@
+make_hgrid:
+  execution: 
+    batchargs: 
+      nodes: 1
+      stdout: /path/to/runscript.out
+      walltime: "08:00:00"
+    executable: /path/make_hgrid
+  config:
+    grid_type: gnomonic_ed
```

Comment:
Example config here remaps the settings from SRW. Several of these are not currently exposed to the user, so I debated configuring special cases, but for now I'm leaving them open